tools/debugger/tests/do_break_generator.expected

```diff
@@ -0,0 +1,25 @@
+Connecting to: localhost:6501
+Connection created!!!
+Stopped at tools/debugger/tests/do_break_generator.js:38
+(escargot-debugger) b do_break_generator.js:21
+Breakpoint 1 at tools/debugger/tests/do_break_generator.js:21 (in f1() at line:20, col:1)
+(escargot-debugger) c
+Stopped at breakpoint:1 tools/debugger/tests/do_break_generator.js:21 (in f1() at line:20, col:1)
+(escargot-debugger) bt
+tools/debugger/tests/do_break_generator.js:21:5 [depth:0] (in f1)
+tools/debugger/tests/do_break_generator.js:38:10
+(escargot-debugger) c
+Stopped at breakpoint:1 tools/debugger/tests/do_break_generator.js:21 (in f1() at line:20, col:1)
+(escargot-debugger) bt
+tools/debugger/tests/do_break_generator.js:21:5 [depth:0] (in f1)
+tools/debugger/tests/do_break_generator.js:25:12 [depth:1] (in f2)
+tools/debugger/tests/do_break_generator.js:38:10
+(escargot-debugger) c
+Stopped at breakpoint:1 tools/debugger/tests/do_break_generator.js:21 (in f1() at line:20, col:1)
+(escargot-debugger) bt
+tools/debugger/tests/do_break_generator.js:21:5 [depth:0] (in f1)
+tools/debugger/tests/do_break_generator.js:25:12 [depth:1] (in f2)
+tools/debugger/tests/do_break_generator.js:29:12 [depth:2] (in f3)
+tools/debugger/tests/do_break_generator.js:38:10
```

Comment:
Let me give you an example about backtrace.
In this case when the last `bt` is triggered, debugger shows recorded backtrace in `next()` call (depth 0 ~ 2) and generator created location.

But when we consider the second approach (showing the real backtrace),
(IMO) debugger will present the following result.
```
tools/debugger/tests/do_break_generator.js:21:5 [depth:0] (in f1)
tools/debugger/tests/do_break_generator.js:25:12 [depth:1] (in f2)
tools/debugger/tests/do_break_generator.js:29:12 [depth:2] (in f3)
tools/debugger/tests/do_break_generator.js:35:?? [depth:3] (in g)
tools/debugger/tests/do_break_generator.js:46:??
```
It shows additional backtrace in generator and `next` method invocation with backtrace in `next()` call as well.
I think that this approach contains more information about generator call info.
Correct me if there is any error in the above result.

If you don't mind, please progress it with this approach. Thanks in advance!

Reply:
I have added a `from != Generator` check, so normal generators are not affected by backtrace recording anymore. However, I kept recording for async generators, and added a test for them. If you want I can exclude them as well.

Regarding the line:
`tools/debugger/tests/do_break_generator.js:35:?? [depth:3] (in g)`

This is currently not working now, because ExecutionPauser does not provide line info. For example:
```
function* g()  {
  throw new Error()
}
g().next()
```
The engine says:
```
Uncaught Error:
test.js (-1:-1)
function next() { [native function] }  (-1:-1)
test.js (68:1)
```
This is unrelated to the debugger, so I don't want to change this in the patch. Actually I don't even know how can it be fixed :)